### ISSUE TITLE
validation accuracy is not increasing

### ISSUE THREAD

**Issue by diliphadoop742** [meownoid/face-identification-tpe#17]
Hello meownoid,

I started training by taking all 4 datasets you mentioned in organize_data.py(lfw,caltech_faces,gt_db,fei), 100 epochs are completed for train_cnn.py but validation accuracy is not increasing it is 0.0046 from first epoch on wards. can you suggest me what changes we need.

**Comment by meownoid** [comment]
Repository is now archived, closing all issues.
